Websites/perf.webkit.org/public/v3/models/triggerable.js

```diff
@@ -52,18 +53,20 @@ class TriggerableConfiguration extends DataModelObject {
         this.#platform = object.platform;
         console.assert(object.supportedRepetitionTypes.every(TriggerableConfiguration.isValidRepetitionType.bind(TriggerableConfiguration)));
         this.#supportedRepetitionTypes = object.supportedRepetitionTypes;
+        this.#testParameters = object.testParameters;
```

Comment:
Nit: Are we guaranteed to always have `object.testParameters` be defined? If not, it may be worth having a fallback of an empty dict here (or in the getter method)

Reply:
Yes, check `v3/models/manifest.js` around line 95.